docs/sources/flow/reference/components/otelcol.auth.basic.md

```diff
@@ -1,5 +1,7 @@
 ---
 canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.auth.basic/
+labels:
+  stage: beta
```

Comment:
Should I make "beta" a link to the [beta](https://grafana.com/docs/agent/latest/stability/#beta) section in the Stability page?